app/models/user.rb

```diff
@@ -83,6 +85,15 @@ def password_reset_expired?
     reset_sent_at < 2.hours.ago
   end
 
+  def feed
+    Micropost.where "user_id = ?", id
+  end
+
+  def display_image
+    size = Settings.size.image.normal
+    image.variant resize_to_limit: [size, size]
```

Comment:
resize image khi hiển thị lên view tránh trường hợp người dùng up ảnh quá to sử dụng minimagick